### ISSUE TITLE
WBN2d is not callable 

### ISSUE THREAD

**Issue by BJTUJia** [mancinimassimiliano/pytorch_wbn#1]
Dear author, thanks a lot for your codes! However, when I call WBN2d in a network to replace nn.Batchnorm2d, during forward process x= WBN2d(x,w), it always reports "forward only takes two inputs, but 3 were given." When I track it down, I found out the problem seems that the class WBN2d can't be called. Any idea what may be the reason?

**Comment by mancinimassimiliano** [comment]
Hi @BJTUJia . Where are the WBN2d layers and their weights w defined? Because if they are wrapped within a network or block, as in ResNet for instance, also the network/block's forward pass must be modified in such a way to take 2 inputs.

**Comment by mancinimassimiliano** [comment]
Hi @BJTUJia . Have you managed to solve the issue?

**Comment by BJTUJia** [comment]
Hi, sorry for the late response. Yes, I have modified the forward pass of the whole block and the issue has been solved. Many thanks for your kind help!

**Comment by mancinimassimiliano** [comment]
Good, you are welcome!
